### ISSUE TITLE
Exception error

### ISSUE THREAD

**Issue by albepal** [py-econometrics/pyfixest#757]
I am performing a regression with two sets of fixed effects with feols and I get an Assertion Error:
<code object demean_model at 0x0000019BB9854BA0, file "./demean_.py", line 8> != <code object demean at 0x0000019BB94A1220, file "./demean_.py", line 187 >
I tried to look for the source of the error in the demean_.py script but I did not find any assert statement.
Does someone know where is the problem coming from?

**Comment by albepal** [comment]
It seems that the error does not arise when I don't use the clustering option. What is weird is that the same command was working until a few weeks ago.

**Comment by s3alfisc** [comment]
Hi @albepal, thanks for reporting!Which version of the package are you using? And could you provide me with a reproducible example ?

**Comment by albepal** [comment]
Hi @s3alfisc, sorry for coming back late on this but it was hard to understand the problem and to be able to have a reproducible example. I understood now there the problem is. 
I am currently using pyfixest 0.27, which requires numba and numpy, among others. I am using numba 0.60 and numpy 2.0.2, and I am coding on VS Code. My problem was that I could run the code from terminal and from the debug console while in debugging mode, but I could not go forward with the debugging as I was receiving the weird error message above. The reason is explained here, and it has nothing to do with pyfixest itself but with numba and the VS Code debugger:

[https://github.com/numba/numba/issues/9753](url)

In other words, if using Python Debugger v2024.12.0, downgrading to numba 0.59.1 works (which implies also having numpy 1.26.0 or lower due to numba requirements).

**Comment by s3alfisc** [comment]
Hi @albepal , thanks for clarifying this and sorry that I did not respond right away!
